changes/760.feature.md

```diff
@@ -0,0 +1,2 @@
+Add support for Ceph file system in Storage Proxy.
+Migrate to mono-repository.
```

Comment:
```suggestion
```
We don't have to tell users that we've migrated this PR to mono repo. Let's remove this line.